src/main/java/lach_01298/qmd/accelerator/LinearAcceleratorLogic.java

```diff
@@ -508,7 +618,9 @@ private void refreshStats()
 				if(componet instanceof TileAcceleratorRFCavity)
 				{
 					TileAcceleratorRFCavity cav = (TileAcceleratorRFCavity) componet;
-					voltage += cav.voltage/8d;
+					if(cavity.is_active) {

```

Comment:
I think it would be better if the everything was left on. This is how it is done with Redstone.